README.md

```diff
@@ -40,6 +40,23 @@ gulp test/typescript
 ```
 gulp test/dotnet
 ```
+#### How to debug the dotnet repo
+For debugging both composed and individual merge state
+```
+autorest input/file --validation --azure-validator --use=path/to/your/repo/sr
+c/dotnet/AutoRest --azure-validator.debugger 
+```
+For debugging composed merge state

```

Comment:
what's the use of debugging composed and/or merged separately?

Reply:
Because there are two threads, one for composed and other for individual. I can be confusing to debug both threads at the same time

Reply:
ok, can you add to the documentation more info on how to choose? and what's the recommendation for debugging the plugin?

Reply:
Fixed